BTVN.B10/main.js

```diff
@@ -0,0 +1,140 @@
+//bai 1: In một bảng nhân của một số bất kỳ với các số từ 1 tới 10 và hiển thị kết quả
+console.log('------------Bài 1---------------')
+function multiplicationTable(x) {
+    for(n = 1; n <= 9; n++)   
+    console.log(x + 'x' + n + '=' + (n*x));
+}
+multiplicationTable(5); 
+
+//bai 2: Hàm nhận tham số là n (số nguyên dương), 1 <= n <= 30 In các số chẵn từ 1 đến n Vd n = 6: in ra: 2 4 6
+console.log('-------------Bài 2--------------')
+function printOutTheNumber(a) {
+    const tong = 0;
+    for( i = 1; i <= a; i++) {
+        if(i % 2 == 0) {
+            console.log(i);
+        }
+    }
+}
+printOutTheNumber(11); 
+
+// bai 3: Hàm nhận tham số là n (số nguyên dương), 1 <= n <= 30 Tính tổng từ 1 đến n vd nhập vào 6: Tổng = 1 + 2 + 3 + 4 + 5 + 6 = 21
+console.log('------------Bài 3---------------')
+function total(n) {
+    let tong = 0;
+    for(i = 0; i <= n; i++) {
+      tong = tong +i;
+    }
+    return tong;
+}
+total(6);
+
+
+//bai 4: Hàm nhận tham số là n (số nguyên dương), 1 <= n <= 30 In ra n! (giai thừa của n)
+console.log('-----------Bài 4----------------')
+function factorial(a) {
+    if(a == 0) {
+        return 1;
+    } else if(a < 0) {
+        console.log('Nhập sai');
+    } else {
+        let giaithua = 1;
+        for(i = 2; i <= a; i++) {
+            giaithua *= i;
+        }
+        return giaithua;
+    }
+}
+factorial(5);
+
+//bai 5: Hàm nhận vào 1 mảng đếm xem trong mảng có bao nhiêu số chẵn Vd: param là [1, 2, 3, 6, 2, 9, 0] thì return 4 vì có 4 số chẵn
+console.log('-----------Bài 5----------------')
+const arrayEvenNumber = (arr = []) => {
+    let arrayOfNumber = 0;
```

Comment:
đặt đơn giản là đếm ban đầu bằng 0 (có 0 số chẵn trong mảng)
```suggestion
    let count = 0;
```